packages/cdk/lib/transcription-service.ts

```diff
@@ -514,5 +522,46 @@ export class TranscriptionService extends GuStack {
 			exportName: `WorkerRoleArn-${props.stage}`,
 			value: workerRole.roleArn,
 		});
+
+		const workerCapacityManagerLambda = new GuLambdaFunction(
```

Comment:
non-blocking: I see guardian CDK has a [`GuScheduledLambda`](https://guardian.github.io/cdk/classes/patterns.GuScheduledLambda.html) which looks like shorthand for the GuLambdaFunction + schedule Rule and might be worth using here if we can? (Not sure it saves us that many lines of code though)

Reply:
ah yes I've even used these before - will do in another PR perhaps in order not to block